update/update.sql

```diff
@@ -395,4 +395,16 @@ ALTER TABLE `mlf2_subscriptions` ADD INDEX `entry` (`eid`);
 INSERT INTO `mlf2_temp_infos` ('name', 'value') VALUES ('access_permission_checks', '0'), ('last_changes', '0'), ('next_daily_actions', '0'), ('version', '2.4.19'); --combined query for relocated settings and for new version string 
 DELETE FROM `mlf2_settings` WHERE name IN('access_permission_checks', 'last_changes', 'next_daily_actions', 'version')
 */
-
+--------------------------------------------------------------------------------
+-- 2.4.19 to 2.4.20
```

Comment:
No, these changes was not part of the update from 2.4.19 to 2.4.20. If accepted they will be part of 2.5 but for now you do not need to document the database changes in this file.